core/src/main/java/com/nuodb/migrator/jdbc/dialect/MariaDBDialect.java

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright (c) 2023, NuoDB, Inc.
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ *
+ *     * Redistributions of source code must retain the above copyright
+ *       notice, this list of conditions and the following disclaimer.
+ *     * Redistributions in binary form must reproduce the above copyright
+ *       notice, this list of conditions and the following disclaimer in the
+ *       documentation and/or other materials provided with the distribution.
+ *     * Neither the name of NuoDB, Inc. nor the names of its contributors may
+ *       be used to endorse or promote products derived from this software
+ *       without specific prior written permission.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
+ * ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+ * WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ * DISCLAIMED. IN NO EVENT SHALL NUODB, INC. BE LIABLE FOR ANY DIRECT, INDIRECT,
+ * INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+ * LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA,
+ * OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF
+ * LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE
+ * OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF
+ * ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+package com.nuodb.migrator.jdbc.dialect;
+
+import com.nuodb.migrator.jdbc.metadata.DatabaseInfo;
+
+import java.sql.SQLException;
+import java.sql.Statement;
+
+
+public class MariaDBDialect extends MySQLDialect {
+
+    public MariaDBDialect(DatabaseInfo databaseInfo) {
+        super(databaseInfo);
+    }
+
+    @Override
+    public void setFetchMode(Statement statement, FetchMode fetchMode) throws SQLException {
+	int fetchSize = fetchMode.isStream() ? 1 : fetchMode.getFetchSize();
+	statement.setFetchSize(fetchSize);
```

Comment:
Nit: please indent these lines.

@adriansuarez Do we have any sort of Java formatter for this project?

Reply:
Not that I'm aware of.